src/backend/distributed/utils/ruleutils_10.c

```diff
@@ -0,0 +1,7745 @@
+/*-------------------------------------------------------------------------
+ *
+ * ruleutils.c
```

Comment:
I very much dislike having the filename inside the file. Mostly useless and annoying to maintain. Will fix.